journal-week-1.md

```diff
@@ -1,10 +1,12 @@
 ##Final Draft
-In my first week here at The Iron Yard aside from getting use to the “emersion” way of learning the areas I had the most difficulty with learning how to use git command lines to push our work to github and how to set up mocha chai for our npm testing. Thankfully with the help of my fellow students and instructors I was able to create two checklists that made doing this a breeze. As requested by the assignment I'm including these from my notes here to make future reference easier.
+In my first week here at The Iron Yard aside from getting use to the “immersion” way of learning the areas I had the most difficulty with learning how to use git command lines to push our work to github and how to set up mocha chai for our npm testing. Thankfully with the help of my fellow students and instructors I was able to create two checklists that made doing this a breeze. As requested by the assignment I'm including these from my notes here to make future reference easier.
 
 Another thing I learned that was new to me was a better understanding of how devleopers work in the job field. Through our Daily Rituals I learned about the Standup Meeting ritual and how real development teams use this to help the team understand how everyone is doing on their assignment. Where are they succeeding and where are they being blocked. I thought this was a neat concept and can't wait to try it out and see how it is implemented in the real world.
 
 Finally I began learning some basic javascript commands such as functions, if/else statements, for loops and arrays and how I can use them in my coding to solve problems. Getting to use them in the homework assignments really helped in my understanding of how they work. For example during lecture when we disscussed functions I had a difficult time wrapping my head around how to use them. But over the weekend after using them over and over again I felt much more competent in my mastery of them. Here's hoping I can do that with arrays and loops in week two!
 
+Helpful check lists I made in week 1:
+
 *Git commands for pushing to github*
 * git status
 * git add filename (to stage it)
```

Comment:
thanks for reflecting! I'd love to hear even more about your experiences with immersion learning.